android/clientmanager/src/main/java/io/mosip/registration/clientmanager/config/SessionManager.java

```diff
@@ -57,6 +57,7 @@ public void saveAuthToken(@NonNull String token) throws Exception {
         editor.putBoolean(IS_DEFAULT, roles.contains("Default"));
         editor.putBoolean(IS_OFFICER, roles.contains("REGISTRATION_OFFICER"));
         editor.apply();
+        return roles;
```

Comment:
As we store complete role details in sharedPreferences, is it required to return the list of roles from here?

Reply:
To be used as it is for now

Reply:
This work will be taken in the next sprint.